contracts/FlowIDTableStaking.cdc

```diff
@@ -1822,6 +1822,91 @@ access(all) contract FlowIDTableStaking {
         return true
     }
 
+    /// Validates that a networking address is properly formatted
+    /// Requirements:
+    /// 1. Must not be an IP address
+    /// 2. Must contain a port number after a colon
+    /// 3. Must be a valid domain name format
+    access(contract) view fun isValidNetworkingAddress(address: String): Bool {
```

Comment:
I feel like this function should be in a string utils contract and not in the staking contract. What do you think? We have one deployed for the bridge that we could potentially put it in

Reply:
Can you point me to it? I can't find the `StringUtils` contract.

Reply:
https://github.com/onflow/flow-evm-bridge/blob/main/cadence/contracts/utils/StringUtils.cdc